docs/SDK/intro.mdx

```diff
@@ -2,10 +2,10 @@
  sidebar_position: 1
 --- 
 
-import LitActionsSection from '../../src/components/SDKCardSections/Actions';
-import AccessControlSection from '../../src/components/SDKCardSections/AccessControl';
-import ResourcesSection from '../../src/components/SDKCardSections/Resources';
-import QuickStartSection from '../../src/components/SDKCardSections/QuickStart';
+import LitActionsSection from '@site/src/components/SDKCardSections/Actions';
```

Comment:
What does `@site` do?

Reply:
@hwrdtm [docusaurus recommends(https://docusaurus.io/docs/versioning#use-absolute-import-within-the-docs) using absolute import when versioning